site/reference/pkg/cat/README.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Cat"
+linkTitle: "cat"
+type: docs
+description: >
+  Print the resources in a file/directory
+---
+
+<!--mdtogo:Short
+    Print the resources in a file/directory
+-->
+
+Cat prints the resources in a file/directory as yaml to stdout.
+
+Cat is useful for printing the resources in a file/directory which might
+contain other non-resource files.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt pkg cat [FILE | DIR]
+
+DIR:
+  Path to a directory with resources. Defaults to the current working directory.
+
+FILE:
+  Path to a resource file.
+```
+
+<!--mdtogo-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+```
+# print resource from a file
+kpt pkg cat path/to/deployment.yaml
+```
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+```
+
+```
+# print resources from current directory
+kpt pkg cat
+```
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+---
+apiVersion: apps/v1
+kind: Service
+metadata:
+  name: nginx-service
+```
+
+<!--mdtogo-->
+
+#### Flags
+
+```sh
+--annotate
+  annotate resources with their file origins.
+
+--format
+  format resource before printing. (default true)
+
+--include-local
+  if true, include local-config in the output.
```

Comment:
What's `local-config`? Do you mean resources with such an annotation? Usage docs have to be precise and have no ambiguity.

Is there a real use case for these two flags?

Reply:
Do you mean resources with such an annotation? => Yes.

I am also not sure if we need them for now. Removed them and we can add later if we need them.

Reply:
Yes, when in doubt, remove flags. They have negative costs by causing confusion and polluting the terminal.